English.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "en_au",
+  "pretty_name": "English",
+  "flag": "https://cdn.nethergames.org/img/flags/australia.png",
+  "translators": ["NetherGames"],
+  "translations": {
+    "command.notinlobby": "That command can only be used in the lobby.",
+    "command.tell.receiver": "{%sender} whispered to you: {%message}",
+    "command.tell.sender": "You whispered to {%receiver}: {%message}",
+    "command.tell.yourself": "You can't send a message to yourself.",
+    "command.vote.slowdown": "§cSlow down, we're checking votes for you already!",
+    "db.error": "An unexpected database error occurred while connecting to the database. Please try again later.",
+    "formhandler.reporter.notreported": "That player can't be reported.",
+    "formhandler.reporter.reported": "You reported {%reported} for {%reason}",
+    "formhandler.reporter.staffmessage": "{%reported} has been reported by {%reporter} for {%reason}",
+    "formhandler.settings.disabled": "Disabled",
+    "formhandler.settings.enabled": "Enabled",
+    "formhandler.settings.fly": "{%status} flying in the lobby.",
+    "formhandler.settings.fly.hidingrank": "You can't fly while you're using a nickname or hiding your rank.",
+    "formhandler.settings.friends": "You are now {%status} friend requests.",
+    "formhandler.settings.guildtag": "You are now {%status} your guild tag.",
+    "formhandler.settings.hideplayers": "You are now {%status} players.",
+    "formhandler.settings.hiding": "hiding",
+    "formhandler.settings.knockback": "{%status} knockback in the lobby.",
+    "formhandler.settings.nick.enabled": "Your name tag has been changed to {%nametag}!",
+    "formhandler.settings.nick.reset": "Reset your name tag.",
+    "formhandler.settings.pet.removed": "Removed your pet.",
+    "formhandler.settings.pet.selected": "Your pet has been changed to a {%pet}.",
+    "formhandler.settings.showing": "showing",
+    "formhandler.switcher.doesntexist": "An account with that username doesn't exist, please try again.",
```

Comment:
Improper grammar. This should be something like, "An account with that username doesn't exist! Please try again."